config/obsoletion.yml

```diff
@@ -0,0 +1,196 @@
+# Configuration of obsolete/deprecated cops used by `ConfigObsoletion`
+
+# Cops that were renamed
+renamed:
+  Layout/AlignArguments: Layout/ArgumentAlignment
+  Layout/AlignArray: Layout/ArrayAlignment
+  Layout/AlignHash: Layout/HashAlignment
+  Layout/AlignParameters: Layout/ParameterAlignment
+  Layout/IndentArray: Layout/FirstArrayElementIndentation
+  Layout/IndentAssignment: Layout/AssignmentIndentation
+  Layout/IndentFirstArgument: Layout/FirstArgumentIndentation
+  Layout/IndentFirstArrayElement: Layout/FirstArrayElementIndentation
+  Layout/IndentFirstHashElement: Layout/FirstHashElementIndentation
+  Layout/IndentFirstParameter: Layout/FirstParameterIndentation
+  Layout/IndentHash: Layout/FirstHashElementIndentation
+  Layout/IndentHeredoc: Layout/HeredocIndentation
+  Layout/LeadingBlankLines: Layout/LeadingEmptyLines
+  Layout/Tab: Layout/IndentationStyle
+  Layout/TrailingBlankLines: Layout/TrailingEmptyLines
+  Lint/DuplicatedKey: Lint/DuplicateHashKey
+  Lint/EndInMethod: Style/EndBlock
+  Lint/HandleExceptions: Lint/SuppressedException
+  Lint/MultipleCompare: Lint/MultipleComparison
+  Lint/StringConversionInInterpolation: Lint/RedundantStringCoercion
+  Lint/UnneededCopDisableDirective: Lint/RedundantCopDisableDirective
+  Lint/UnneededCopEnableDirective: Lint/RedundantCopEnableDirective
+  Lint/UnneededRequireStatement: Lint/RedundantRequireStatement
+  Lint/UnneededSplatExpansion: Lint/RedundantSplatExpansion
+  Naming/UncommunicativeBlockParamName: Naming/BlockParameterName
+  Naming/UncommunicativeMethodParamName: Naming/MethodParameterName
+  Style/DeprecatedHashMethods: Style/PreferredHashMethods
+  Style/MethodCallParentheses: Style/MethodCallWithoutArgsParentheses
+  Style/OpMethod: Naming/BinaryOperatorParameterName
+  Style/SingleSpaceBeforeFirstArg: Layout/SpaceBeforeFirstArg
+  Style/UnneededCapitalW: Style/RedundantCapitalW
+  Style/UnneededCondition: Style/RedundantCondition
+  Style/UnneededInterpolation: Style/RedundantInterpolation
+  Style/UnneededPercentQ: Style/RedundantPercentQ
+  Style/UnneededSort: Style/RedundantSort
+
+# Cops that were moved to a different department
+moved:
```

Comment:
Do we need `renamed` and `moved` separately only to provide better messages?  In general it's easy to infer from the old name/new name that a cop has been renamed or moved, so I'm wondering if we should not simplify this.

Reply:
I've merged the two together now, thanks for the suggestion!